PyOpenMS 3.2.0 build fails on bioconda due to Eigen3 not found

Related PR: https://github.com/bioconda/bioconda-recipes/pull/51014

conda build log: https://dev.azure.com/bioconda/bioconda-recipes/_build/results?buildId=71098&view=logs&j=e14e69ff-a0ae-55c4-b71d-229b239cfb2f&t=7df82132-b284-504b-53d6-7d3e63519572&l=583

Error:
>12:33:44 BIOCONDA INFO (OUT) CMake Error at /opt/conda/conda-bld/pyopenms_1728390567959/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/cmake/OpenMS/OpenMSTargets.cmake:78 (set_target_properties):
12:33:44 BIOCONDA INFO (OUT)   The link interface of target "OpenMS" contains:
12:33:44 BIOCONDA INFO (OUT) 
12:33:44 BIOCONDA INFO (OUT)     Eigen3::Eigen
12:33:44 BIOCONDA INFO (OUT) 
12:33:44 BIOCONDA INFO (OUT)   but the target was not found.  Possible reasons include:
12:33:44 BIOCONDA INFO (OUT) 
12:33:44 BIOCONDA INFO (OUT)     * There is a typo in the target name.
12:33:44 BIOCONDA INFO (OUT)     * A find_package call is missing for an IMPORTED target.
12:33:44 BIOCONDA INFO (OUT)     * An ALIAS target is missing.
12:33:44 BIOCONDA INFO (OUT) 
12:33:44 BIOCONDA INFO (OUT) Call Stack (most recent call first):
12:33:44 BIOCONDA INFO (OUT)   /opt/conda/conda-bld/pyopenms_1728390567959/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/cmake/OpenMS/OpenMSConfig.cmake:111 (include)
12:33:44 BIOCONDA INFO (OUT)   CMakeLists.txt:39 (find_package)

Error related to this:
https://github.com/OpenMS/OpenMS/blob/f47d834cdd692f90aaa9e336a28a0220b5fa3e2c/src/pyOpenMS/CMakeLists.txt#L39

Comment underneath:
> #TODO when separating pyopenms from OpenMS, specify a pyopenms version and the compatible OpenMS version to be searched above.
> For now we just hope that the right OpenMS version from this same git repo is installed and found and
> therefore give pyopenms here the same version. CF_OPENMS_PACKAGE_VERSION is used in env.py for setup.py

Could it be that the hard coded link
https://github.com/jonasscheid/bioconda-recipes/blob/4c2eace48dfdd6efc2f70a419b26058ae2b6cd88/recipes/pyopenms/meta.yaml#L14

is causing that `that the right OpenMS version from this same git repo is` NOT found?

--- a/cmake/OpenMSConfig.cmake.in
+++ b/cmake/OpenMSConfig.cmake.in
@@ -50,6 +50,7 @@ endif()
 #TODO somehow add the same/compatible versions that were found by OpenMS? And what about static vs dynamic? E.g. if we link to static zlib in OpenMS(.dll) what (if at all) can/should the consumer link against?
 find_dependency(Qt5 @QT_MIN_VERSION@ COMPONENTS @OpenMS_QT_COMPONENTS@)
 find_dependency(XercesC)
+find_dependency(Eigen3 3.3.4)
 find_dependency(LIBSVM 2.91)
 
 # Rest are private linked libraries
@@ -58,7 +59,6 @@ find_dependency(LIBSVM 2.91)
 #find_dependency(GLPK)
 #find_dependency(ZLIB)
 #find_dependency(BZip2)
-#find_dependency(Eigen3 3.3.4)
 #find_dependency(SQLite3 3.15.0)
 #find_dependency(HDF5)
 

